code/modules/clothing/glasses/glasses.dm

```diff
@@ -286,7 +286,7 @@
 
 /obj/item/clothing/glasses/red
 	name = "red glasses"
-	desc = "A sweet pair of red shades."
+	desc = "Hey, you're looking good, senpai!"

```

Comment:
https://github.com/tgstation/tgstation/pull/22109/files

It IS in there, not sure why it shows as a difference